azure/durable_functions/orchestrator.py

```diff
@@ -146,13 +146,11 @@ def _reset_timestamp(self):
         decision_started_events = [e_ for e_ in self.durable_context.histories
                                    if e_.event_type == HistoryEventType.ORCHESTRATOR_STARTED
                                    and e_.timestamp > last_timestamp]
-        if len(decision_started_events) == 0:
-            self.durable_context.current_utc_datetime = None
-        else:
-            self.durable_context.decision_started_event = \
-                decision_started_events[0]
-            self.durable_context.current_utc_datetime = \
-                self.durable_context.decision_started_event.timestamp
+        if len(decision_started_events) != 0:
+            self.durable_context.decision_started_event = decision_started_events[0]
+
+        self.durable_context.current_utc_datetime = \
```

Comment:
I think this effectively doesn't do anything if `len(decision_started_events) == 0`. If that is the case, I would prefer we move this inside of the if block.